src/maps/factory.ts

```diff
@@ -139,10 +139,14 @@ function getOffset(i: number, length: number, startsLower: boolean): number {
   return total - placement;
 }
 
-export function town(townName: string): LandData {
+export function town(townName: string, goods?: Good | Good[]): LandData {
```

Comment:
Rather than seeding cubes this way, the pattern I've seen for most other maps is to override the starter, especially for the somewhat common pattern of adding starting cubes to every town. Take a look at `starter.ts` for chesapeake-and-ohio as an example.